homeassistant/components/reolink/number.py

```diff
@@ -62,10 +62,22 @@ class ReolinkNumberEntityDescription(
         native_step=1,
         get_min_value=lambda api, ch: api.zoom_range(ch)["focus"]["pos"]["min"],
         get_max_value=lambda api, ch: api.zoom_range(ch)["focus"]["pos"]["max"],
-        supported=lambda api, ch: api.zoom_supported(ch),
+        supported=lambda api, ch: api.supported(ch, "zoom"),
         value=lambda api, ch: api.get_focus(ch),
         method=lambda api, ch, value: api.set_zoom(ch, int(value)),
     ),
+    ReolinkNumberEntityDescription(
+        key="floodlight_brightness",
+        name="Floodlight brightness",
+        icon="mdi:spotlight-beam",
+        mode=NumberMode.SLIDER,
```

Comment:
The slider is automatically set in general, was it needed to override it in this case?

../Frenck